dls-controls/malcolmjs
# Feature/layout model #94

## Description

Wired up the layout attribute to display blocks in the middle panel. It only displays the blocks currently, there is only very minimal interaction capability.

## Testing instructions

Add a set up instructions describing how the reviewer should test the code
- Review code
- Check Travis build
- Review changes to test coverage
- Run `npm start` and navigate to http://localhost:3000/gui/PANDA/layout/PANDA:CLOCKS

## Agile board tracking

connect to #94
